### ISSUE TITLE
Advantages?

### ISSUE THREAD

**Issue by mikekscholz** [fmalekpour/OctoPrint-Gpioshutdown#2]
This seems a little overkill code-wise, when the same thing can be achieved by adding dtoverlay=gpio-shutdown (defaults to placing a switch between gpio3/pin5 and GND/pin6) and dtparam=pwr_led_gpio=27 to your /boot/config.txt... Am I missing something?

**Comment by fmalekpour** [comment]
You are right, it's just another way to do that with some GUI settings in Octoprint. Also I like the LED to be on when Octoprint is up and running, not just the os.

**Comment by SirHaziel** [comment]
Just a quick question.
How to wake it up again using the same button? Is that coded and I'm missing or was it intended to be a shutdown only?

**Comment by fmalekpour** [comment]
You can't wake up the pi with code. But you can wire another switch to reset/run pin and use it to wake the pi up.

**Comment by rafaelmartins** [comment]
it would be nice if the plugin could avoid shutdown during print, or require a different activation pattern to shutdown during print, like "press for 5 seconds". this would be a huge advantage over the dtoverlay approach.
